### ISSUE TITLE
2025-06-06, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#192]
# Today's Arxiv Papers

(1) [SparseMM: Head Sparsity Emerges from Visual Concept Responses in MLLMs](http://arxiv.org/abs/2506.05344v1), 2025-06-05

(2) [Inference-Time Hyper-Scaling with KV Cache Compression](http://arxiv.org/abs/2506.05345v1), 2025-06-05

(3) [Why LLM Safety Guardrails Collapse After Fine-tuning: A Similarity Analysis Between Alignment and Fine-tuning Datasets](http://arxiv.org/abs/2506.05346v1), 2025-06-05

(4) [Direct Numerical Layout Generation for 3D Indoor Scene Synthesis via Spatial Reasoning](http://arxiv.org/abs/2506.05341v1), 2025-06-05

(5) [VideoMolmo: Spatio-Temporal Grounding Meets Pointing](http://arxiv.org/abs/2506.05336v1), 2025-06-05

(6) [Search Arena: Analyzing Search-Augmented LLMs](http://arxiv.org/abs/2506.05334v1), 2025-06-05

(7) [Unleashing Hour-Scale Video Training for Long Video-Language Understanding](http://arxiv.org/abs/2506.05332v1), 2025-06-05

(8) [MINT-CoT: Enabling Interleaved Visual Tokens in Mathematical Chain-of-Thought Reasoning](http://arxiv.org/abs/2506.05331v1), 2025-06-05

(9) [Improving Data Efficiency for LLM Reinforcement Fine-tuning Through Difficulty-targeted Online Data Selection and Rollout Replay](http://arxiv.org/abs/2506.05316v1), 2025-06-05

(10) [Constrained Entropic Unlearning: A Primal-Dual Framework for Large Language Models](http://arxiv.org/abs/2506.05314v1), 2025-06-05

(11) [Time to Talk: LLM Agents for Asynchronous Group Communication in Mafia Games](http://arxiv.org/abs/2506.05309v1), 2025-06-05

(12) [ProRefine: Inference-time Prompt Refinement with Textual Feedback](http://arxiv.org/abs/2506.05305v1), 2025-06-05

(13) [Perceive Anything: Recognize, Explain, Caption, and Segment Anything in Images and Videos](http://arxiv.org/abs/2506.05302v1), 2025-06-05

(14) [Power Law Guided Dynamic Sifting for Efficient Attention](http://arxiv.org/abs/2506.05300v1), 2025-06-05

(15) [Sample Complexity and Representation Ability of Test-time Scaling Paradigms](http://arxiv.org/abs/2506.05295v1), 2025-06-05

(16) [EOC-Bench: Can MLLMs Identify, Recall, and Forecast Objects in an Egocentric World?](http://arxiv.org/abs/2506.05287v1), 2025-06-05

(17) [Micro-Act: Mitigate Knowledge Conflict in Question Answering via Actionable Self-Reasoning](http://arxiv.org/abs/2506.05278v1), 2025-06-05

(18) [From Play to Replay: Composed Video Retrieval for Temporally Fine-Grained Videos](http://arxiv.org/abs/2506.05274v1), 2025-06-05

(19) [Teaming in the AI Era: AI-Augmented Frameworks for Forming, Simulating, and Optimizing Human Teams](http://dx.doi.org/10.1145/3699682.3727574), 2025-06-05

(20) [LeanPO: Lean Preference Optimization for Likelihood Alignment in Video-LLMs](http://arxiv.org/abs/2506.05260v1), 2025-06-05

(21) [CLATTER: Comprehensive Entailment Reasoning for Hallucination Detection](http://arxiv.org/abs/2506.05243v1), 2025-06-05

(22) [SECNEURON: Reliable and Flexible Abuse Control in Local LLMs via Hybrid Neuron Encryption](http://arxiv.org/abs/2506.05242v1), 2025-06-05

(23) [MonkeyOCR: Document Parsing with a Structure-Recognition-Relation Triplet Paradigm](http://arxiv.org/abs/2506.05218v1), 2025-06-05

(24) [LLM-First Search: Self-Guided Exploration of the Solution Space](http://arxiv.org/abs/2506.05213v1), 2025-06-05

(25) [The Common Pile v0.1: An 8TB Dataset of Public Domain and Openly Licensed Text](http://arxiv.org/abs/2506.05209v1), 2025-06-05

(26) [RELIC: Evaluating Compositional Instruction Following via Language Recognition](http://arxiv.org/abs/2506.05205v1), 2025-06-05

(27) [Transformers Meet In-Context Learning: A Universal Approximation Theory](http://arxiv.org/abs/2506.05200v1), 2025-06-05

(28) [Quantifying Cross-Modality Memorization in Vision-Language Models](http://arxiv.org/abs/2506.05198v1), 2025-06-05

(29) [MokA: Multimodal Low-Rank Adaptation for MLLMs](http://arxiv.org/abs/2506.05191v1), 2025-06-05

(30) [TreeRPO: Tree Relative Policy Optimization](http://arxiv.org/abs/2506.05183v1), 2025-06-05

(31) [On the Comprehensibility of Multi-structured Financial Documents using LLMs and Pre-processing Tools](http://arxiv.org/abs/2506.05182v1), 2025-06-05

(32) [Qwen3 Embedding: Advancing Text Embedding and Reranking Through Foundation Models](http://arxiv.org/abs/2506.05176v1), 2025-06-05

(33) [ECoRAG: Evidentiality-guided Compression for Long Context RAG](http://arxiv.org/abs/2506.05167v1), 2025-06-05

(34) [Dissecting Bias in LLMs: A Mechanistic Interpretability Perspective](http://arxiv.org/abs/2506.05166v1), 2025-06-05

(35) [Do Large Language Models Judge Error Severity Like Humans?](http://arxiv.org/abs/2506.05142v1), 2025-06-05

(36) [DiCoRe: Enhancing Zero-shot Event Detection via Divergent-Convergent LLM Reasoning](http://arxiv.org/abs/2506.05128v1), 2025-06-05

(37) [Membership Inference Attacks on Sequence Models](http://arxiv.org/abs/2506.05126v1), 2025-06-05

(38) [The NTNU System at the S&I Challenge 2025 SLA Open Track](http://arxiv.org/abs/2506.05121v1), 2025-06-05

(39) [DIMCIM: A Quantitative Evaluation Framework for Default-mode Diversity and Generalization in Text-to-Image Generative Models](http://arxiv.org/abs/2506.05108v1), 2025-06-05

(40) [Interpretable Multimodal Framework for Human-Centered Street Assessment: Integrating Visual-Language Models for Perceptual Urban Diagnostics](http://arxiv.org/abs/2506.05087v1), 2025-06-05

(41) [Parking, Perception, and Retail: Street-Level Determinants of Community Vitality in Harbin](http://arxiv.org/abs/2506.05080v1), 2025-06-05

(42) [LLM-Guided Scenario-based GUI Testing](http://arxiv.org/abs/2506.05079v1), 2025-06-05

(43) [Just a Scratch: Enhancing LLM Capabilities for Self-harm Detection through Intent Differentiation and Emoji Interpretation](http://arxiv.org/abs/2506.05073v1), 2025-06-05

(44) [RIVAL: Reinforcement Learning with Iterative and Adversarial Optimization for Machine Translation](http://arxiv.org/abs/2506.05070v1), 2025-06-05

(45) [Reason-to-Recommend: Using Interaction-of-Thought Reasoning to Enhance LLM Recommendation](http://arxiv.org/abs/2506.05069v1), 2025-06-05

(46) [Does It Make Sense to Speak of Introspection in Large Language Models?](http://arxiv.org/abs/2506.05068v1), 2025-06-05

(47) [Debatable Intelligence: Benchmarking LLM Judges via Debate Speech Evaluation](http://arxiv.org/abs/2506.05062v1), 2025-06-05

(48) [A Survey on Vietnamese Document Analysis and Recognition: Challenges and Future Directions](http://arxiv.org/abs/2506.05061v1), 2025-06-05

(49) [TALL -- A Trainable Architecture for Enhancing LLM Performance in Low-Resource Languages](http://arxiv.org/abs/2506.05057v1), 2025-06-05

(50) [Automatic Robustness Stress Testing of LLMs as Mathematical Problem Solvers](http://arxiv.org/abs/2506.05038v1), 2025-06-05

(51) [Hierarchical Language Models for Semantic Navigation and Manipulation in an Aerial-Ground Robotic System](http://arxiv.org/abs/2506.05020v1), 2025-06-05

(52) [FinMultiTime: A Four-Modal Bilingual Dataset for Financial Time-Series Analysis](http://arxiv.org/abs/2506.05019v1), 2025-06-05

(53) [Controlling Summarization Length Through EOS Token Weighting](http://arxiv.org/abs/2506.05017v1), 2025-06-05

(54) [ComfyUI-Copilot: An Intelligent Assistant for Automated Workflow Development](http://arxiv.org/abs/2506.05010v1), 2025-06-05

(55) [QiMeng: Fully Automated Hardware and Software Design for Processor Chip](http://arxiv.org/abs/2506.05007v1), 2025-06-05

(56) [SCOP: Evaluating the Comprehension Process of Large Language Models from a Cognitive View](http://arxiv.org/abs/2506.05000v1), 2025-06-05

(57) [Mathematical Reasoning for Unmanned Aerial Vehicles: A RAG-Based Approach for Complex Arithmetic Reasoning](http://arxiv.org/abs/2506.04998v1), 2025-06-05

(58) [BacPrep: An Experimental Platform for Evaluating LLM-Based Bacalaureat Assessment](http://arxiv.org/abs/2506.04989v1), 2025-06-05

(59) [FPTQuant: Function-Preserving Transforms for LLM Quantization](http://arxiv.org/abs/2506.04985v1), 2025-06-05

(60) [TextVidBench: A Benchmark for Long Video Scene Text Understanding](http://arxiv.org/abs/2506.04983v1), 2025-06-05

(61) [Agentic AI for Intent-Based Industrial Automation](http://arxiv.org/abs/2506.04980v1), 2025-06-05

(62) [Evaluating Prompt-Driven Chinese Large Language Models: The Influence of Persona Assignment on Stereotypes and Safeguards](http://arxiv.org/abs/2506.04975v1), 2025-06-05

(63) [From Struggle (06-2024) to Mastery (02-2025) LLMs Conquer Advanced Algorithm Exams and Pave the Way for Editorial Generation](http://arxiv.org/abs/2506.04965v1), 2025-06-05

(64) [PoCGen: Generating Proof-of-Concept Exploits for Vulnerabilities in Npm Packages](http://arxiv.org/abs/2506.04962v1), 2025-06-05

(65) [APVR: Hour-Level Long Video Understanding with Adaptive Pivot Visual Information Retrieval](http://arxiv.org/abs/2506.04953v1), 2025-06-05

(66) [Simulating LLM-to-LLM Tutoring for Multilingual Math Feedback](http://arxiv.org/abs/2506.04920v1), 2025-06-05

(67) [When Thinking LLMs Lie: Unveiling the Strategic Deception in Representations of Reasoning Models](http://arxiv.org/abs/2506.04909v1), 2025-06-05

(68) [Verbose ListOps (VLO): Beyond Long Context -- Unmasking LLM's Reasoning Blind Spots](http://arxiv.org/abs/2506.04907v1), 2025-06-05

(69) [From Objects to Anywhere: A Holistic Benchmark for Multi-level Visual Grounding in 3D Scenes](http://arxiv.org/abs/2506.04897v1), 2025-06-05

(70) [ICPC-Eval: Probing the Frontiers of LLM Reasoning with Competitive Programming Contests](http://arxiv.org/abs/2506.04894v1), 2025-06-05

(71) [Evaluating the Effectiveness of Linguistic Knowledge in Pretrained Language Models: A Case Study of Universal Dependencies](http://arxiv.org/abs/2506.04887v1), 2025-06-05

(72) [LLMs for sensory-motor control: Combining in-context and iterative learning](http://arxiv.org/abs/2506.04867v1), 2025-06-05

(73) [Adapting Online Customer Reviews for Blind Users: A Case Study of Restaurant Reviews](http://arxiv.org/abs/2506.04865v1), 2025-06-05

(74) [Sparse Autoencoders, Again?](http://arxiv.org/abs/2506.04859v1), 2025-06-05

(75) [Prompting LLMs: Length Control for Isometric Machine Translation](http://arxiv.org/abs/2506.04855v1), 2025-06-05

(76) [Multiple-Choice Question Generation Using Large Language Models: Methodology and Educator Insights](http://dx.doi.org/10.1145/3631700.3665233), 2025-06-05

(77) [On Automating Security Policies with Contemporary LLMs](http://arxiv.org/abs/2506.04838v1), 2025-06-05

(78) [OpenMaskDINO3D : Reasoning 3D Segmentation via Large Language Model](http://arxiv.org/abs/2506.04837v1), 2025-06-05

(79) [Joint Evaluation of Answer and Reasoning Consistency for Hallucination Detection in Large Reasoning Models](http://arxiv.org/abs/2506.04832v1), 2025-06-05

(80) [From EHRs to Patient Pathways: Scalable Modeling of Longitudinal Health Trajectories with LLMs](http://arxiv.org/abs/2506.04831v1), 2025-06-05

(81) [A Reasoning-Based Approach to Cryptic Crossword Clue Solving](http://arxiv.org/abs/2506.04824v1), 2025-06-05

(82) [Evaluating Vision-Language and Large Language Models for Automated Student Assessment in Indonesian Classrooms](http://arxiv.org/abs/2506.04822v1), 2025-06-05

(83) [LogicPuzzleRL: Cultivating Robust Mathematical Reasoning in LLMs via Reinforcement Learning](http://arxiv.org/abs/2506.04821v1), 2025-06-05

(84) [Dissecting Logical Reasoning in LLMs: A Fine-Grained Evaluation and Supervision Study](http://arxiv.org/abs/2506.04810v1), 2025-06-05

(85) [Towards LLM-Centric Multimodal Fusion: A Survey on Integration Strategies and Techniques](http://arxiv.org/abs/2506.04788v1), 2025-06-05

(86) [MMSU: A Massive Multi-task Spoken Language Understanding and Reasoning Benchmark](http://arxiv.org/abs/2506.04779v1), 2025-06-05

(87) [Fine-Grained Interpretation of Political Opinions in Large Language Models](http://arxiv.org/abs/2506.04774v1), 2025-06-05

(88) [Identifying Reliable Evaluation Metrics for Scientific Text Revision](http://arxiv.org/abs/2506.04772v1), 2025-06-05

(89) [GOLFer: Smaller LM-Generated Documents Hallucination Filter & Combiner for Query Expansion in Information Retrieval](http://arxiv.org/abs/2506.04762v1), 2025-06-05

(90) [Exp4Fuse: A Rank Fusion Framework for Enhanced Sparse Retrieval using Large Language Model-based Query Expansion](http://arxiv.org/abs/2506.04760v1), 2025-06-05

(91) [Truth in the Few: High-Value Data Selection for Efficient Multi-Modal Reasoning](http://arxiv.org/abs/2506.04755v1), 2025-06-05

(92) [Multi-Layer GRPO: Enhancing Reasoning and Self-Correction in Large Language Models](http://arxiv.org/abs/2506.04746v1), 2025-06-05

(93) [Lifelong Evolution: Collaborative Learning between Large and Small Language Models for Continuous Emergent Fake News Detection](http://arxiv.org/abs/2506.04739v1), 2025-06-05

(94) [Evaluation is All You Need: Strategic Overclaiming of LLM Reasoning Capabilities Through Evaluation Design](http://arxiv.org/abs/2506.04734v1), 2025-06-05

(95) [Beyond Accuracy: Dissecting Mathematical Reasoning for LLMs Under Reinforcement Learning](http://arxiv.org/abs/2506.04723v1), 2025-06-05

(96) [SPARTA ALIGNMENT: Collectively Aligning Multiple Language Models through Combat](http://arxiv.org/abs/2506.04721v1), 2025-06-05

(97) [Towards Holistic Visual Quality Assessment of AI-Generated Videos: A LLM-Based Multi-Dimensional Evaluation Model](http://arxiv.org/abs/2506.04715v1), 2025-06-05

(98) [LLM-based phoneme-to-grapheme for phoneme-based speech recognition](http://arxiv.org/abs/2506.04711v1), 2025-06-05

(99) [Empowering Economic Simulation for Massively Multiplayer Online Games through Generative Agent-Based Modeling](http://arxiv.org/abs/2506.04699v1), 2025-06-05

(100) [On the Mechanism of Reasoning Pattern Selection in Reinforcement Learning for Language Models](http://arxiv.org/abs/2506.04695v1), 2025-06-05
